### PR TITLE
travis: test more go versions

### PR DESCRIPTION
**Pull request by vbatts**
`tip` is failing https://travis-ci.org/vbatts/tar-split/jobs/210745272

Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 language: go
 go:
   - tip
-  - 1.6.2
-  - 1.5.4
+  - 1.8
+  - 1.7
+  - 1.6
+  - 1.5
```
Comment (tianon) [inline .travis.yml]: If you add `.x` to the end of these, Travis will resolve that to the latest minor release of each version (ie, `1.8.x` will be just `1.8`, but `1.7.x` will be `1.7.5`)

Comment (tianon) [inline .travis.yml]: (see https://github.com/travis-ci/travis-build/blob/5e3ef60b0d43b8ef56fb6fa77ba3269fbf945fa6/lib/travis/build/config.rb#L54-L70)

**Comment by vbatts** [comment]
Oh good to know. I had assumed the short version would just do that.

**Comment by vbatts** [comment]
it's a monotonic time. Work around, like so :point_right:  https://github.com/golang/go/commit/0e3355903d2ebcf5ee9e76096f51ac9a116a9dbb

**Comment by vbatts** [comment]
fixed it

**Pull request merged**